### PR TITLE
Adding table method clickFilterButton()

### PR DESCRIPTION
**Pull request by aweng98**
Use value of attribute `data-test-state` when click table filter button for page synchronization. 

HOW:
Adding a new method `clickFilterButton` in Table.scala to ensure `awaitReady()` call is executed after change in state of `loading` is detected. Because value changes from `ready` to `loading` to `ready`.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala [hunk 1]
```diff
@@ -20,20 +20,22 @@ trait Clickable extends LazyLogging { this: Component =>
     } catch {
       case e: TimeoutException =>
         // show me query string on failed WebElement
-        new WebDriverWait(webDriver, 5).until(ExpectedConditions.elementToBeClickable(query.element.underlying))
+        new WebDriverWait(webDriver, 1).until(ExpectedConditions.elementToBeClickable(query.element.underlying))
```
Comment (jacmrob) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala:23]: why reducing the wait time?

**Review by jacmrob — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala (hunk 1 above)
Comment (aweng98) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala:23]: I didn't see need for longer wait to terminate test. It has no useful purpose. thank you for review.

**Comment by aweng98** [comment]
to be sure. 

jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Pull request merged**